osquery/tables/system/posix/authorized_keys.cpp

```diff
@@ -48,13 +68,78 @@ void genSSHkeysForUser(const std::string& uid,
       logger.log(google::GLOG_WARNING, s.getMessage());
       continue;
     }
+
     // Protocol 1 public key consist of: options, bits, exponent, modulus,
     // comment; Protocol 2 public key consist of: options, keytype,
     // base64-encoded key, comment.
     for (const auto& line : split(keys_content, "\n")) {
       if (!line.empty() && line[0] != '#') {
-        Row r = {{"uid", uid}, {"key", line}, {"key_file", keys_file.string()}};
-        r["pid_with_namespace"] = "0";
+        auto splitted_line = split(line, " ");
```

Comment:
As I understand the underlying format, it's `options, keytype, base64-encoded key, comment` the first being optional. Does it make sense to limit the number of splits here?

Reply:
Please notice that the number of space/whitespace is unknown because the first parameter(options) and last parameter(comment) can contain whitespace for example:
`command="/usr/bin/tinyfugue hostname" ssh-rsa AAAAB3NzaC1yc2EAAA...OFy5Lwc8Lo+Jk= host @ name .com`
So it can't be limited to a fixed number of splits.